backend/battles/tests/test_models.py

```diff
@@ -0,0 +1,21 @@
+from django.test import TestCase
+
+from model_mommy import mommy
+
+from battles.models import Team
+
+
+class TestTeamModel(TestCase):
+    def setUp(self):
+        self.battle = mommy.make("battles.Battle")
+        self.trainer = mommy.make("users.User")
+        self.pokemon_set = mommy.prepare("pokemon.Pokemon", _quantity=3, make_m2m=True)
```

Comment:
This file is also breaking on CI. The error is:
```
TypeError: Pokemon() got an unexpected keyword argument 'make_m2m'
```

I don't think that the `make_m2m` attribute exists for `prepare`. Check the documentation of the model mommy: https://model-bakery.readthedocs.io/en/latest/basic_usage.html#more-than-one-instance